### PR TITLE
Footer 

### PR DESCRIPTION
**Pull request by HaltonO**
Footer links to about us and contact us pages

### CODE REVIEW

File: imports/ui/Splash/Splash/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
```
Comment (madeline-petersen) [inline imports/ui/Splash/Splash/index.js]: It looks like we have both `Splash/Splash/index.js` and `Splash/index.js`. Do you want to pick the one you want to use and have it live at `Splash/index.js`?  Delete the other one.

**Comment by madeline-petersen** [comment]
You'll have to solve these conflicts before merging... Do:

```
git checkout master
git pull
git checkout footer_
git merge origin/master
```

And resolve any conflicts that come up.

File: imports/ui/AboutUs/index.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        <h4 id="header">About us:</h4>
+        <p id="paragraph">Students tend to frequent a limited set of places, namely their place of residence and classes.
+         The people who share their interests are not necessarily in these same places.
+          We intend SmirkSpace to provide a place for people with like interests to connect.
+          To this end, we are targeting niche groups in the UVic community; people whose interests are rare
+          enough that they won’t necessarily find congenial company without intentionally seeking it out.</p>
+          <h4>Our Team:</h4>
+          <table >
+           <tr>
+             <td><img width="200" height="200" src="/images/tim.png" /><p>Tim Baker</p></td>
```
Comment (madeline-petersen) [inline imports/ui/AboutUs/index.js:18]: Love these 😄

**Comment by HaltonO** [comment]
I fixed those.. I hope it works now :) let me know

**Comment by madeline-petersen** [comment]
Let's make sure there aren't any changes to package.json...

```
git checkout footer_
git checkout master -- package.json
```

(This pulls master's version of package.json onto your branch)

**Comment by madeline-petersen** [comment]
screen shot 2016-11-17 at 3 54 36 pm

Looks like we are deleting an image and adding the same one back in?  Maybe run the following to pull in the copy from master...

```
git checkout master -- public/images/starsbackground.jpg
```

File: imports/ui/Footer/index.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import { Meteor } from 'meteor/meteor';
+import { Link } from 'react-router'
+import './index.css';
+
+export default class Footer extends Component {
```
Comment (madeline-petersen) [inline imports/ui/Footer/index.js]: Let's write this as a pure function...

```
export default function Footer() {
  return (
    <div>
      <footer className="footer"><Link to="/about-us" className="link1">About us</Link> &nbsp; | &nbsp;
      <Link to="/contact-us" className="link1">Contact us</Link></footer>
    </div>
  );
}
```

File: imports/ui/Footer/index.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import { Meteor } from 'meteor/meteor';
```
Comment (madeline-petersen) [inline imports/ui/Footer/index.js]: Remove this line, Meteor isn't being used.

File: imports/ui/Footer/index.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
```
Comment (madeline-petersen) [inline imports/ui/Footer/index.js]: We won't need `, { Component }` when Footer is written as a pure function (see below)

File: imports/ui/Footer/index.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+import { Meteor } from 'meteor/meteor';
+import { Link } from 'react-router'
```
Comment (madeline-petersen) [inline imports/ui/Footer/index.js]: This line needs a semi-colon (";")

**Pull request merged**